docs/UserGuide.adoc

```diff
@@ -220,6 +220,115 @@ Examples:
 * `gradebook assign c/CS2040C i/A0168111 g/Assignment 1 m/10` +
 Assigns 10 marks to CS2040C student with matric number A0168111.
 
+=== Class Management
+==== Creating a class
```

Comment:
The 'command' should be listed alongside this subheader, and for the other subheaders.

Reply:
Alright. Thanks.